tests/test_eof.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from dahuffman.huffmancodec import _EOF
+
+
+@pytest.fixture(params=(
+    b"\0",
+    b"abc",
+    "abc",
+    0,
+    100,
+    -1000,
+    False,
+    True,
+    None,
+    [],
+    (),
+    {},
+    [-100, 0, 10],
+    (-100, 10))
+)
+def to_compare(request):
```

Comment:
What I meant in a previous comment was pytest allows you to define the fixture as a method of `TestEndOfFileSymbol` (instead of a module level function), which makes it available only to other methods of that class